src/data-sources/salesforce-d2c/SalesforceD2CSettings.tsx

```diff
@@ -54,34 +54,36 @@ export const SalesforceB2CSettings = ( {
 				displayName={ state.display_name ?? '' }
 				handleOnChange={ handleOnChange }
 				heading={ {
-					icon: SalesforceCommerceB2CIcon,
+					icon: SalesforceCommerceD2CIcon,
 					width: '100px',
 					height: '75px',
 					verticalAlign: 'middle',
 				} }
-				inputIcon={ SalesforceCommerceB2CIcon }
+				inputIcon={ SalesforceCommerceD2CIcon }
 				uuid={ uuid }
 			>
 				<TextControl
 					type="text"
-					label={ __( 'Merchant shortCode', 'remote-data-blocks' ) }
-					onChange={ shortCode => {
-						handleOnChange( 'shortcode', shortCode ?? '' );
+					label={ __( 'Instance URL', 'remote-data-blocks' ) }
```

Comment:
Is there a better way to get the instance url? Note that I haven't done any validation here to ensure it's `https` for example. I don't quite know the format of these urls, hence why I've given an example only.

Reply:
> Is there a better way to get the instance url?

Do you mean in React / UI terms? Or in API terms? I don't understand the question, sorry.

Reply:
I think he's talking about validation for the "Instance URL" input:

![image](https://github.com/user-attachments/assets/51aefbc2-db1b-4c76-a3cd-e66308773c87)

I think the example is adequate and we don't need a lot of field validation. However, it would be great if we could have some sort of full-page validation on the bottom of this page to confirm all fields work with a test request, something like we have for the Google Sheets connection:

![sheets-connection-confirmation](https://github.com/user-attachments/assets/b09f7239-15c3-483e-a8f7-be254788485f)

That way we can make sure there aren't any problems before we save the connection details.

Reply:
I didn't add that as a todo on the PR but I've that on my list before this PR is finalized.

@chriszarate  - I should have made that clearer that I did mean validation of the url once it's inputted by the user.

Reply:
Implemented this in